### PR TITLE
Implement IsFixedLength API

### PR DESCRIPTION
**Pull request by ajcvickers**
Issue #9579

### CODE REVIEW

**Comment by ajcvickers** [comment]
Added both @smitpatel and @bricelam as reviews since Brice is out, but really only need one.

File: src/EFCore.Relational/Migrations/MigrationBuilder.cs [hunk 1]
```diff
@@ -60,6 +60,44 @@ public MigrationBuilder([CanBeNull] string activeProvider)
         /// <param name="defaultValueSql"> The SQL expression to use for the column's default constraint. </param>
         /// <param name="computedColumnSql"> The SQL expression to use to compute the column value. </param>
         /// <returns> A builder to allow annotations to be added to the operation. </returns>
+        [Obsolete("Use the overload with the most parameters.")]
```
Comment (bricelam) [inline src/EFCore.Relational/Migrations/MigrationBuilder.cs]: These shouldn't be obsolete. They could be generated by Migrations if you using every parameter but the new one. (On `ColumnsBuilder` too.)

Comment (ajcvickers) [reply src/EFCore.Relational/Migrations/MigrationBuilder.cs]: @bricelam Okay. Will un-obsolete them. Is there a better non-breaking pattern we could use here?

Comment (bricelam) [reply src/EFCore.Relational/Migrations/MigrationBuilder.cs]: No, this is the best non-breaking pattern, and we can still remove the "old" overload when binary breaks are allowed. We just can't mark them obsolete.

**Pull request merged**